apps/mull-ui/src/app/pages/create-event/date-calendar/date-calendar.scss

```diff
@@ -1,6 +1,7 @@
 @import 'variables';
 
 .date-calendar-container {
+  width: 100%;
```

Comment:
I added `width 100%` to each class separately. It made more sense since `.custom-text-input-container` already exists in the file and has another style applied to it as well.

Reply:
Aight sounds good

Reply:
I see you added the styling to each class in create.event.scss. However now this line you added is dead code since its applied twice in two places. Please remove it.